edxval/tests/test_api.py

```diff
@@ -2410,10 +2410,12 @@ def test_update_video_image(self):
         """
         Verify that `update_video_image` api function works as expected.
         """
-        self.assertEqual(self.course_video.video_image.image.name, self.image_url)
-        self.assertEqual(self.course_video2.video_image.image.name, self.image_url2)
-        self.assertEqual(ImageFile(open(self.image_path1, 'rb')).size, ImageFile(open(self.image_url, 'rb')).size)
-        self.assertEqual(ImageFile(open(self.image_path2, 'rb')).size, ImageFile(open(self.image_url2, 'rb')).size)
+        image_url = self.image_url.lstrip('/')
```

Comment:
Tests were failing due to extra / being appended before local test files path.

From Django 3.1 Release Notes

> The STATIC_URL and MEDIA_URL settings set to relative paths are now prefixed by the server-provided value of SCRIPT_NAME (or / if not set). This change should not affect settings set to valid URLs or absolute paths.